guides/common/modules/proc_setting-an-alternative-directory-for-remote-execution-jobs-in-push-mode.adoc

```diff
@@ -0,0 +1,36 @@
+[id="Setting_an_Alternative_Directory_for_Remote_Execution_Jobs_in_Push_Mode_{context}"]
+= Setting an alternative directory for remote execution jobs in push mode
+
+.Procedure
+. On your {ProjectServer} or {SmartProxy}, create a new directory:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# mkdir _/My_Remote_Working_Directory_
```

Comment:
The parameter is the working directory for non-pull mode on target machines. There's no need to create the directory on the server

Reply:
@adamruzicka can you confirm this?

Reply:
Confirmed via chat.